libs/market-info/src/components/market-info/info-market.tsx

```diff
@@ -129,16 +130,26 @@ export const Info = ({ market, onSelect }: InfoProps) => {
     {
       title: t('Market price'),
       content: (
-        <MarketInfoTable
-          data={pick(
-            market.data,
-            'name',
-            'markPrice',
-            'bestBidPrice',
-            'bestOfferPrice'
-          )}
-          decimalPlaces={market.decimalPlaces}
-        />
+        <>
+          <MarketInfoTable
+            data={{
+              ...pick(
+                market.data,
+                'name',
+                'markPrice',
+                'bestBidPrice',
+                'bestOfferPrice'
+              ),
+              quoteUnit: market.tradableInstrument.instrument.product.quoteName,
+            }}
+            decimalPlaces={market.decimalPlaces}
+          />
+          <p className="text-xs">
+            {t(
```

Comment:
non-blocking: this can be used like so:
```
t('The notional value of a position of size 1 is 1 settlement asset (%s) for every 1 quote unit (%s).', [assetSymbol, quoteUnit])
```
and in such a way can be possible to translate